docs/faq.mdx

```diff
@@ -18,6 +18,15 @@ This documentation is relevant to the new IBM Quantum&reg; Platform. If you need
 
 For FAQs specific to the execution modes, see the [Qiskit Runtime execution modes FAQ](/guides/execution-modes-faq) guide.
 
+## Trouble creating an IBM Cloud account
+
+If you get an error when you create an IBM Cloud&reg; account, first check whether you have access to the [Support center.](https://cloud.ibm.com/unifiedsupport/supportcenter)
+
+- If you do have access, click on "create case", selecting *Account* as the Topic and *Account verification* as the Subtopic.
+- If you do not have access, contact Support by calling 1-866-325-0045 and selecting the third option.
```

Comment:
Will this work for anyone in the world?

Reply:
The number matches what they have in the [public docs](https://cloud.ibm.com/docs/account?topic=account-ts_accnt_inactive&locale=en), and I don't see any region-specific phone numbers related to Support.